core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -1880,6 +1880,7 @@ object KafkaZkClient {
                   case _ => null
                 }
                 SetDataResponse(resultCode, setDataOp.path, ctx, stat, responseMetadata)
+              case zkOp => throw new IllegalStateException(s"Unexpected zkOp: $zkOp")
```

Comment:
@junrao Please check that this is OK.

Reply:
@ijuma : Yes, this part looks good to me.